home work 21/script.js

```diff
@@ -23,12 +29,6 @@ ul.addEventListener("click", (event) => {
   }
 });
```

Comment:
this should be wrapped in separate function (i.e. selectRange). Also try to simplify this logic, it could be implemented with one loop (forEach) only